src/dmd/expression.d

```diff
@@ -7096,6 +7104,42 @@ extern (C++) final class FileInitExp : DefaultInitExp
     }
 }
 
+/***********************************************************
+ */
+extern (C++) final class ArgnameInitExp : DefaultInitExp
```

Comment:
Now there are three different names in play here: `ArgnameInitExp`, `__ARG_STRING__` and `argumentName`. Pick one of them and use it consistently. Please also avoid abbreviations.

Reply:
changed to more consistent naming:
```
__ARG_STRING__
argString
ArgStringInitExp
isArgStringInitExp
resolveArgString
```
I did use commonly used abbreviation though, `__ARGUMENT_STRING__` would be too long IMO (and some ppl had suggesed ARG also)